repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/InspectPreviousBuild.java

```diff
@@ -0,0 +1,49 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import fr.inria.spirals.jtravis.entities.Build;
+import fr.inria.spirals.jtravis.entities.BuildStatus;
+import fr.inria.spirals.jtravis.helpers.BuildHelper;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector4Bears;
+
+/**
+ * Created by fermadeiral.
+ */
+public class InspectPreviousBuild extends AbstractStep {
+
```

Comment:
I don't really see the point of this step right now? Can't it be merge with TestPreviousBuild?